test/docker/ubuntu2004

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+
+RUN apt-get update
+RUN apt-get install -y build-essential cmake git uuid-dev libgnutls28-dev libfaketime
+RUN apt-get install -y python
```

Comment:
When building the image locally, I had the issue that `tzdata` installation (likely as dependency) required manual intervention. This can be circumvented by setting 
```
DEBIAN_FRONTEND="noninteractive" 
```
in the corresponding `RUN` line.

Reply:
Will change

Reply:
This is the only outstanding issue here, otherwise this looks good to merge.

Reply:
Needs an update here.